src/screens/DiscoveryScreenV2/fragment/NewsFragment.js

```diff
@@ -1,161 +1,140 @@
 import RenderItem from '../elements/RenderItem';
 import share from '../../../utils/share';
 import useIsReady from '../../../hooks/useIsReady';
-import {Context} from '../../../context/Store';
-import {fonts} from '../../../utils/fonts';
-import {getUserId} from '../../../utils/users';
-import {newsDiscoveryContentParamBuilder} from '../../../utils/navigation/paramBuilder';
-import {withInteractionsManaged} from '../../../components/WithInteractionManaged';
-import {COLORS} from '../../../utils/theme';
+import { Context } from '../../../context/Store'
+import { colors } from '../../../utils/colors';
+import { fonts } from '../../../utils/fonts';
+import { getUserId } from '../../../utils/users';
+import { newsDiscoveryContentParamBuilder } from '../../../utils/navigation/paramBuilder';
+import { withInteractionsManaged } from '../../../components/WithInteractionManaged';
 
 const NewsFragment = ({
-  isLoadingDiscoveryNews = false,
-  isFirstTimeOpen,
-  setSearchText = () => {},
-  setIsFirstTimeOpen = () => {},
-  news = []
+    isLoadingDiscoveryNews = false,
+    isFirstTimeOpen,
+    setSearchText = () => { },
+    setIsFirstTimeOpen = () => { },
+    news = []
 }) => {
-  const [myId, setMyId] = React.useState('');
-  const [defaultNews] = React.useContext(Context).news;
-
-  const isReady = useIsReady();
-
-  const navigation = useNavigation();
-
-  React.useEffect(() => {
-    const parseToken = async () => {
-      const id = await getUserId();
-      if (id) {
-        setMyId(id);
-      }
-    };
-    parseToken();
-  }, []);
-
-  // React.useEffect(() => {
-  //     if(news.length > 0) setIsFirstTimeOpen(false)
-  // }, [news])
-
-  const renderNewsItem = () => {
-    if (isFirstTimeOpen) {
-      return [<DiscoveryTitleSeparator key="news-title-separator" text="Suggested News" />].concat(
-        defaultNews.news.map((item, index) => {
-          const onContentClicked = () => {
-            navigation.navigate('DetailDomainScreen', {
-              item: {
-                id: item.id,
-                score: item.domain.credderScore,
-                follower: 0
-              }
-            });
-          };
-
-          // Disable on press content if view should be navigated to LinkContextScreen
-          if (!item.dummy)
-            return (
-              <RenderItem
-                key={`news-screen-item-${index}`}
-                item={item}
-                selfUserId={myId}
+    const [myId, setMyId] = React.useState('')
+    const [defaultNews] = React.useContext(Context).news
+
+    const isReady = useIsReady()
+
+    const navigation = useNavigation()
+
+    React.useEffect(() => {
+        const parseToken = async () => {
+            const id = await getUserId();
+            if (id) {
+                setMyId(id);
+            }
+        };
+        parseToken();
+    }, []);
+
+    // React.useEffect(() => {
+    //     if(news.length > 0) setIsFirstTimeOpen(false)
+    // }, [news])
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out useEffect hook might be a leftover from debugging or development. If it's not needed, consider removing it to keep the code clean.



<!-- This is an auto-generated comment by OSS CodeRabbit -->